ONSdigital/response-operations-ui
# Remove address logging

# Motivation and Context

A previous PR introduced some log lines which log social respondents' addresses and iac, these need to be removed

# What has changed



* Removed addresses and iac from log lines

# How to test?



* Check none of the items being logged can contain address details
* Run the service on this branch with log level `DEBUG` and navigate to view a social case details page while tailing the logs, you should see `case_id` and `status` in the debug logs but no address details

# Links



https://github.com/ONSdigital/response-operations-ui/pull/224
https://trello.com/c/WTjuaOv5/190-sus004-generate-a-new-access-code-iac-for-an-lms-case
